### ISSUE TITLE
kill helmfile during "helm dep up" sometimes prevent next "apply" to work

### ISSUE THREAD

**Issue by sgandon** [roboll/helmfile#541]
when lanching a `helmfile apply` this will do a `helm dependencies update`. If this process is killed it sometimes leave a folder called `\tmpcharts`.
The problem is that when  you launch the `apply ` again the `helm dep up` command will fail with
```
Error: Unable to move current charts to tmp dir: rename /mychart/charts /mychart/tmpcharts: file exists
```

**Comment by mumoshu** [comment]
Thanks for reporting!

This seems like a bug in helm. Would it be possible for you to open an issue in upstream?

https://github.com/helm/helm/blob/53d432fa58748412ff3dc10bc27cbf996d96c3ed/pkg/downloader/manager.go#L190

**Comment by sgandon** [comment]
yes I can try again but I already opened a few tickets on helm and even provided fixes through PRs but no one really cared.

**Comment by mumoshu** [comment]
Mind sharing urls to those PRs?

**Comment by sgandon** [comment]
here it is, and guess what after one year someone did a review (yey !).
https://github.com/helm/helm/pull/3987

**Comment by sgandon** [comment]
here is the related issue I created in the `helm` repo : 
https://github.com/helm/helm/issues/5567
